### PR TITLE
sci-libs/lapack: add deprecated functions also to 3.9.0

### PR DESCRIPTION
**Pull request by efferre79**
as commented in https://bugs.gentoo.org/702254#c6

### CODE REVIEW

File: sci-libs/lapack/lapack-3.9.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="BLAS,CBLAS,LAPACK,LAPACKE reference implementations"
+HOMEPAGE="http://www.netlib.org/lapack/"
```
Comment (vaukai) [inline sci-libs/lapack/lapack-3.9.0-r1.ebuild:9]: https://netlib.org/lapack/

Comment (efferre79) [reply sci-libs/lapack/lapack-3.9.0-r1.ebuild]: updated

**Comment by thesamesam** [comment]
Thanks! Merged in 681412c77f8c1d58962f8159eb0cd0f648888d70

**Comment by juippis** [comment]
Please note that in future you need to sign-off using your legal name, not a nickname.
https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin

(I see you use it in Bugzilla so should be no problem)

**Comment by thesamesam** [comment]
Yeah, I wasn't that bothered because the changes are actually from Debian. This is just about adding the patch. I could've just dropped the signed-off-by line but I thought I'd give some credit for doing the PR.

**Pull request closed without merging**